.gitignore

```diff
@@ -15,6 +15,13 @@ storybook/public/
 # JUnit report files
 reports
 
+# OS generated files
+.DS_Store
+.DS_Store?
+.Trashes
+ehthumbs.db
+Thumbs.db
+
```

Comment:
This was a drive-by, because I was seeing macOS things try to sneak into my commits, and the thumbs.db tend to sneak in similarly when I am on my desktop back home.

Reply:
Hmm... So far we've avoided adding editor / OS-specific files like these, and instead have each person use their own global gitignore to avoid files specific to their environment from being added. That said, I don't feel strongly about this.

@hswolff Do you feel strongly about it?

Reply:
Nope, looks grrrreat!